translations/test/TranslationServer.js

```diff
@@ -1093,8 +1093,49 @@ describe('TranslationServer', function () {
         assert.equal(response.statusCode, '200');
         done();
       });
-    });
+      it('drop default tags by default', function(done) {
+        var osm = server.handleInputs({
+            translation: 'MGCP',
+            method: 'POST',
+            path: '/translateFrom',
+            osm: "<osm version='0.6' generator='JOSM'><node id='-161534' lat='38.9078407955' lon='-77.03267545044'><tag k='FCODE' v='AL015' /> <tag k='CCN' v='UNK' /></node></osm>"
+        });
+
+        xml2js.parseString(osm, function(err, result) {
+          assert.equal(result.osm.node[0].tag[0].$.k, 'building')
+          assert.equal(result.osm.node[0].tag[0].$.v, 'yes')
```

Comment:
Not sure if order ever changes, but maybe assert that tag length is 1 here, and then 2 below where UNK is excpected?